lib/camerapreview.dart

```diff
@@ -379,6 +393,13 @@ class CameraAwesomeState extends State<CameraAwesome> with WidgetsBindingObserve
     });
   }
 
+  _initVideoSize() {
+    widget.videoSize.addListener(() async {
+      await CamerawesomePlugin.setVideoSize(
+          widget.videoSize.value.width.toInt(), widget.videoSize.value.height.toInt());
+    });
+  }
```

Comment:
1. Should this listener be removed somewhere in the lifecycle?
2. Is this listener really required? It looks to me that the video size is only set once, so it may be less error prone to just update the video size there immediately.